src/style.css

```
@@ -0,0 +1,114 @@
+@font-face {
+  font-family: "Avenir";
+  src: url(./Fonts/Avenir/Avenir.ttf) format("truetype");
+  font-weight: normal;
+  font-style: normal;
+}
+
+.search-bar {
+  width: 100%;
+  height: 70px;
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  -webkit-box-sizing: border-box;
+          box-sizing: border-box;
+  -webkit-box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+          box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  font-family: Avenir, sans-serif;
+  font-size: 16px;
+  line-height: 22px;
+  color: #3d4e61;
+  padding-left: 62px;
+}
+
+.search-bar::-webkit-input-placeholder {
+  font-family: Avenir, sans-serif;
+  font-size: 16px;
+  line-height: 22px;
+  color: #3d4e61;
+}
+
+.search-bar:-ms-input-placeholder {
+  font-family: Avenir, sans-serif;
+  font-size: 16px;
+  line-height: 22px;
+  color: #3d4e61;
+}
+
+.search-bar::-ms-input-placeholder {
+  font-family: Avenir, sans-serif;
+  font-size: 16px;
+  line-height: 22px;
+  color: #3d4e61;
+}
+
+.search-bar::placeholder {
+  font-family: Avenir, sans-serif;
+  font-size: 16px;
+  line-height: 22px;
+  color: #3d4e61;
+}
+
+.search-bar:hover {
+  -webkit-box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+          box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+}
+
+.search-bar:focus {
+  background: -webkit-gradient(linear, left top, left bottom, from(#fff), to(#f6f6f7));
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  border-radius: 3px;
+  font-weight: 700;
+  outline: none;
+}
+
+.search-bar-form {
+  position: relative;
+}
+
+.search-bar-form::after {
+  position: absolute;
+  top: 25px;
+  left: 26px;
+  content: "";
+  width: 19px;
+  height: 19px;
+  background-image: url(./images/Search.svg);
+}
+
+.search-bar--sm {
+  font-size: 14px;
+  line-height: 16px;
+  height: 42px;
+  padding-left: 33px;
+}
+
+.search-bar--sm::-webkit-input-placeholder {
+  font-size: 14px;
+  line-height: 16px;
+}
+
+.search-bar--sm:-ms-input-placeholder {
+  font-size: 14px;
+  line-height: 16px;
+}
+
+.search-bar--sm::-ms-input-placeholder {
+  font-size: 14px;
+  line-height: 16px;
+}
+
+.search-bar--sm::placeholder {
+  font-size: 14px;
+  line-height: 16px;
+}
+
+.search-bar-form--sm::after {
+  top: 15px;
+  left: 13px;
+  width: 11px;
+  height: 11px;
+  background-size: cover;
+}
+/*# sourceMappingURL=style.css.map */
```

document must end with one empty line
```suggestion
/*# sourceMappingURL=style.css.map */

```